vehicle/SUV.h

```diff
@@ -5,8 +5,8 @@
 
 class SUV final: public Car {
 public:
-    SUV(IEngine* engine, IGear* gear)
-        : Car(engine, gear) {}
+    SUV(IEngine* m_engine, IGear* m_gear)
```

Comment:
same here

Reply:
updated, p_ for parameters, m_ for class members